### ISSUE TITLE
[Bug]  mapwin/buffered.py, AttributeError: module 'time' has no attribute 'clock'

### ISSUE THREAD

**Issue by marcocurreli** [OSGeo/grass#536]
**Errors wxpython after GUI is opened**

> Traceback (most recent call last):
>   File ".../gui/wxpython/mapwin/buffered.py", line 640, in OnSize
>     self.resize = time.clock()

- Operating System: Linux 
- GRASS GIS version 7.8.1 and 7.8.2
- Python version: 3.8.2

Changing that line as follows in my machine it works fine.
file buffered.py.diff:
```
640c640
<        self.resize = time.clock()
---
>         self.resize = time.perf_counter()
```

**Comment by neteler** [comment]
Here is some related function which might be useful (?):

https://github.com/OSGeo/grass/blob/b74dfd2f5045864dab6037d54a1d610b278417d9/lib/python/script/utils.py#L456

```
"""
    Return time counter to measure performance for chunks of code.
    Uses time.clock() for Py < 3.3, time.perf_counter() for Py >= 3.3.
...
```

**Comment by petrasovaa** [comment]
This is already fixed in 78 development version.

**Comment by wenzeslaus** [comment]
Thanks @marcocurreli. Just to be clear, the fix will available in 7.8.3 which will be released soon.
